specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-02-14-privatepreview/postgresql.json

```diff
@@ -1250,18 +1337,39 @@
       ],
       "description": "Represents a server."
     },
+    "MaintenanceWindow": {
```

Comment:
How about the old definitions for CustomerMaintenanceWindow or CustomerMaintenanceWindowList? will you remove them?

Reply:
Removed the Unused entity definitions